\documentclass[12pt]{amsart}

\addtolength{\hoffset}{-2.25cm}
\addtolength{\textwidth}{4.5cm}
\addtolength{\voffset}{-2.5cm}
\addtolength{\textheight}{5cm}
\setlength{\parskip}{0pt}
\setlength{\parindent}{15pt}

\usepackage{amsthm}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage[colorlinks = true, linkcolor = black, citecolor = black, final]{hyperref}

\usepackage{graphicx}
\usepackage{multicol}
\usepackage{ marvosym }
\usepackage{wasysym}
\newcommand{\ds}{\displaystyle}


\pagestyle{myheadings}

\setlength{\parindent}{0in}

\pagestyle{empty}

\begin{document}

\thispagestyle{empty}

{\scshape VETM 690} \hfill {\scshape \Large  Ch. 7: Mutation} \hfill {\scshape Mar 7 2021}
\medskip
\hrule
\bigskip

{\large \bf Estimation of Mutation in Quantitative Traits:}
\begin{itemize}
\item Mutation: the source of population genetic variation on which natural or aritifical selection acts (to produce genetic changes leading to adaptive evolution)
\item Number of rounds of replicaiton from zygote to the formation of gametes varies between species and is why rates of mutations vary between species
\item Men and women: the number of rounds of replication in ova stay at 22 while it continues to grow for spermatozoa in men
\item However, chromosomal mutations increase in women because of cytoplasmic deterioration in oocytes over time
\item Given the difficulty of identifying genotypes, it is only possible to obtain average estimates of the effect of mutations and their degree of dominance
\item Mutational variance $V_M$ is the increase in additive genetic variance per generation due to mutation
\item And thus mutational heritability is the ratio of $V_M/V_E$ 
\item Probability of fixation of a mutation:
\begin{itemize}
\item In finite populations, genetic drift may cause beneficial alleles to be lost and deleterious alleles to be fixed 
\item If the mutational selection coefficient is small ($N_es/N < 1$)and $N_es$ is high, then the probability of fixation can be approximated by the selective advantage s, so that most beneficial mutations of small effect are lost in a population of finite size
\item Neutral mutations have a $P_f$ equal to their initial frequency $1/2N$
\item Deleterious mutations are destined to be eliminated by selection 
\item Mutations where absolute value of $N_e$ is substantially smaller than 1, the probability of fixation is the same for that of a neutral mutation. When mutation is advantageous or disadvantageous, the action of drift is more intense than selection and so the fate of the mutation is dependent on chance. This is referred to as quasi-neutral mutations. 
\item Mutations can be beneficial, neutral or deleterious based on effective population size
\end{itemize}
\item Estimating the rate of mutation and mutational effects
\begin{itemize}
\item Balanced chromosome technique: cross an individual with another with a balanced chromosome (carrying inversions that inhibit recombinations along with visible markers for identification) 
\item Cross wt male with If female to obtain copies of chromosome that are used to create multiple independent lines
\item Mutations accumulate in each line and disappear when deleterious heterozygous effects are sufficiently large
\item This approach has been applied to C. elegans, where lines are started by either self-fertilization or brother sister mating. Mutations accumulate in the genome, and because $N_es << 1$, mutations behave as neutral and have a highly probability of fixation. 
\item Mutations can be studied if they affect traits that are fitness components, either directly measured (productivity) or it may be in competition with the starting population (founder individuals in C. elegans can be frozen then thawed and compared with individuals from a line)
\item Accumulating mutations is expected to reduce the mean value of the trait will increasing the variance in that trait, and $\delta V/ \delta M =\bar{s}(1+C^2)$, where U is the haploid genome mutation rate, and C is the coefficient of variation of mutational effects, and s is the average of selective effects of mutations weighted by their mutation rate (Bateman-Mukai method)
\item Bateman-Mukai enables the estimation of mutational variance (increase in additive genetic variance per generation due to new mutations) and mutational heritability 
\item Past experiments typically show that most mutations are of small effects, and this is studied using the gamma distribution
\item U has been estimated for multiceullular eukaryotes (U = 0.08) and viruses (0.93). 
\item Mean mutational coefficient of variation for fitness traits is $CV_M = 0.026$ 
\item The Bateman-Mukai method produces estimates with bias of unknown magnitude, because we do not know the distribution of mutational effects 
\item rate of deleterious mutation can also be estimated from molecular data as $U = U_n\phi$, where $U_n$ is the rate of nucleotide mutation per generation for the genome and phi is the fraction of mutations whose deleterious effect is large enough for selection to determine its elimination, calculated by $\phi = 1 - \pi_g/\pi_s$ where $pi_s$ is the rate substitution rate of mutations that are assumed to be exclusively neutral, and $pi_g$ is the total divergence observed between to twin species 
\item Detecting mutation rates using molecular methods only discovers mutations of large effect, and those of small effect are not found because of insufficient power
\end{itemize}
\item The dominance coefficient
\begin{itemize}
\item Mutation accumulation experiments enable the estimates of average coefficient of dominance
\item First estimate is obtained as the ratio between the value of the trait from a cross between two lines along with the sum of the values of the trait in those lines. Mutations will be found in heterozygosis in the cross
\item The decline in mean of trait divded by the sum of declines occuring in lines makes up the ratio estimates, which is a measure of the average of dominance coefficients weighted by selection coefficients
\item Unbiased method for estimating average coefficient of dominance: regress the trait values at cross between lines of heterozygotes onto homozygotes (acting as the sum of trait values in parental lines)
\item Joint distribution of selection and dominance coefficients: hard to do because the effect of each mutation must be estimated individually 
\item Large effect mutations in homozygosis are assumed to be recessive
\item Small-effect mutations are additive 
\item Fisher hypothesis: a large amount of observed mutations are recessive. It is thought that they were initially additive and that dominance modifiers appeared at other loci that would cause the heterozygoye expression to become similar to wild homozygous 
\item More common explanation: Wright's physiological theory of dominance. This is based on the idea that the enzymatic activity of heterozygotes is intermediate between homozygotes, but the flux of a metabolic pathway with several enzymes is non linear. But if the mutation has a high activity-reducing effect, then the heterozygote is more similar to the wild homozygote, so the mutation has recessive gene action. This explains why large-effect mutations tend to be recessive
\end{itemize}
\item Beneficial mutations and summary of mutational parameters: there is a large variation in mutational parameters between estimates for different traits, species and estimation methods
\item Somatic mutation rates may be 50x higher than those of the germline
\item Mutations and environmental factors: can be additive or multiplicative. Can also be synergistic epistasis (greater than combining them) or antagonistic (lower than combining)
\item Mutations can be conditionally deleterious (such as temperature dependent ones)
\item Rate of mutation can also be contingent on the genetic background and whether a population is adapted to its environment. For instance, populations with low average fitness see higher rates of beneficial mutation compared to those with normal average fitness (though these are compensatory) 
\item Mutational parameters for quasi-neutral traits: can be studied using mutation accumulation or, starting with a population lacking genetic variability, artifical selection
\item If the trait is neutral, then variability is determined exclusively by mutation (increase) or drift (reduction) 
\item Under the neutral model, the heterozygosity in the mutation-drift equilibrium is a function of $N_e$ and mutation rate per locus
\item Alternatively, knowing heterozygosity and mutation rate we can estimate effective population size
\item Additive genetic variance in each generation is the result of the decrease in drift and the increase by new mutation, so equilibrium will be reached 
\item Though traits are not generally neutral, the following expression can be used if $N_e$ is sufficiently small: $\hat{H} = \frac{4N_eu}{4N_eu + 1}$
\item The genetic variance between mutation accumulation lines increases at a rate $2 \delta FV_A$
\item The response to artificial selection due to mutation can be expressed as $R_t = t\hat{V}i/\sqrt{V_P}$ where $i$ is the intensity with which selection is applied and $V_P$ is the phenotypic variance in that trait 
\item Genes with low effect have effect sizes that are indistinguishable from additivity, but those of large effect are usually recessive and have deleterious pleiotropic effects on fitness 
\item Implications of deleterious mutations in populations of large sizes: the maximum possible frequency of a deleterious allele at the mutation selection balance is $\hat{q} = u/sh$ 
\item Small effect of alleles on heterozygosis implies a substantial reduction of equilibrium frequency
\item Mutation load: decrease in mean fitness of a population compared to that with no segregating deleterious mutations 
\item The mutation load for a recessive allele is equal to u which implies that in deleterious mutation selection equilibrium, the load only depends on mutation rate regardless of frequency
\item Joint effect of all loci with presumed deleterious alleles: $L \approx 1 - e^{-2U}$ 
\item The mean number of mutations eliminated per individual by selection is L(z-y) where L is the load, z is the mean number of del. mutations in individuals and the y is the mean number of mutations carried by survivors 
\item The number of mutations eliminated by selection may be greater than expected due to synergistic epistasis
\item Non-panmictic populations: equilibrium load decreases as the inbreeding coefficient increases (showing how inbreeding purges deleterious mutations)
\item Hard selection: fitness directly translates into survival probability, quite unrealistic in terms of the amount of progeny needed for survival
\item But mutation load refers to population fitness relative to the mutation-free genotype which is often non-existent in natural populations  so the issue with hard selection is reduced/disappears (so long as fitness is considered in relative terms)
\item Relaxation of selection in human population: possible accumulation of deleterious mutations
\item Attempted eugenics in 1970s over the world was a failure due to the ineffective elimination of recessive alleles which are masked in heterozygotes 
\item Consider loads outside of mutation: load due to segregation of heterozygotes in overdominant fitness models, recombination, advantageous allelic recombination, or environemntal heterogeneity or inbreeding
\item Estimating the average dominance coefficient in mutation-selection equilibrium: using balanced chromosome designs as mentioned before. Homozygous individuals can be obtained for a chromosome, and carriers of chromosomes heterozygous for all loci. Then regress the heterozygotes on homozygotes to estimate the harmonic mean of h weighted by s, given by: $\hat{b} =  \frac{\bar{s}}{\bar{s/h}}$
\item Estimating mutational parameters in equilibrium from panmictic and inbred populations: if data on means and variances of fitness are available, then we can compute the mutation rate, mean selection coefficient of mutations and dominance coefficients
\item Mutation and recombination: sex is one of the most important evolutionary aspects for the fixation of beneficial (or deleterious) mutations
\item Sex carries a two-fold cost assuming an equal number of descendants per individual
\item Fisher and Muller proposed the evolutionary advantage of recombination: the probability of fixing beneficial mutations at different loci will be independent with recombination
\item Fixation of beneficial mutations is much faster in populations with sex and mutation, providing a net selective advantage
\item Muller expanded on deleterious mutations. They can be assembled inthe same genome and thus simultaneously eliminated. In asexual species, mutations accumulate without genome reconstruction, so the minimum number of mutations carried by a given genome grows irreversibly (Muller's ratchet) 
\item The fixation probability of mutations for any rate of recombination can be obtained from an extension of the formula for predicting $N_e$ under linkage, which is $N_e \approx\frac{-U}{s + (L/2)}$
\item Hill-Robertson effect: when two alleles of different loci segregate, interference can occur so their probability of fixation is smaller than that if they did not segregate simultaenously (occurs in the absence of LD or through interaction between loci)
\item In chapter 5 we saw that selecting for a loci increases the magnitude of drift on an unselected, neutral locus due to random associations, reducing $N_e$ ascribed to the neutral locus 
\item Hill-Robertson is an extension of this when two loci are linked. The association between loci will persist over more generations and will be diluted by recombination, progressivly reducing $N_e$
\item If two linked genes increased in frequency due to selection $N_e$ decreases but the effect is reduced if recombination is present
\item Hill-Robertson effect also predicts an increase in the rate of fixating deleterious mutations if no recombination is present 
\end{itemize}

{\large \bf Problems}   
\begin{enumerate}
\item In a population with $N = 100$ individuals and $N_e = 50$, an additive mutation occurs with favourable effect $s = 0.1$ in the homozygote. (a) What is the probability that the mutation will end up being fixed in the population? (b) What would be that probability if the mutation were recessive? (c) If the mutation were deleterious and additive, what would be the mean persistance time and the average number of copies present in the population prior to elimination? 
\begin{gather*}
P_f = \frac{1 - e^{-s(N_e/N}}{1-e^{-2N_es}} = \frac{1 - e^{-0.1(50/100)}}{1-e^{-2(50)(0.1)}} = 0.049 \\
P_f \approx \frac{\sqrt{2N_es/\pi}}{N} = \frac{\sqrt{2(50)0.1/\pi}}{100} = 0.018\\
N_es = 50(-0.1) = -5 << -1 ; P_f = 0 ; h = -0.5 \\ 
t = 2[ln(-1/sh)+1-\gamma = 2[ln(-1/0.1(-0.5) + 1 - 0.5772] = 6.84 \\
-1/sh = -1/(0.1(-0.5)) = 20
\end{gather*}
\item In a mutation accumulation experiment where 100 crossbred brother-sister lines of Drosophila melanogaster were used, the mean viabilities initially and in geneation 100 were 0.8 and 0.35, and the corresponding variances between lines 0.05 and 0.14, respectively. If the distribution of effects of mutations is assumed to be exponential, what are the estimates that can be obtained for the genomic mutation rate, U, the mean effect of mutations $\bar{s}$ and the mutational variance $V_M$? \\
\begin{gather*}
\triangle M = (v_0 - v_f)/t = (0.8 - 0.35)/100 = 0.0045\\
\triangle V = (V_0 - V_f)/t = (0.14-0.05)/100 = 0.0009\\
C^2/ = 1/\beta = 1/1\\
\frac{\triangle V }{\triangle M} = \bar{s}(1 + C^2) \\
\frac{\triangle V }{2\triangle M} = \bar{s} = 0.0009/(2(0.0045)) = 0.1\\
U = \frac{(1+C^2)\triangle M^2}{\triangle V} = \frac{2(0.0045)^2}{0.0009}=0.045\\
V_M = \triangle V /2 = 0.0009 /2 = 0.000045    
\end{gather*}
\item A strain of Drosophila melanogaster devoid of genetic variation was created by means of crosses using balanced chromosomes. With this strain, lines were founded with effective size $N_e = 20$ and maintained for 100 generations in panmixia. We then evaluate a neutral quantitative trait whose mutational variance has been estimated to be $V_M = 0.0005$. What would be the approximate expected value of the variance between lines and the additive variance of the trait in the last generation?
\begin{gather*}
V_{B,t} = 2tV_M = 2(100)(0.0005)=0.1\\
V_A\approx 2N_eV_M=2(20)0.0005=0.02\\
V_{A(t)} = V_{A(t-1)}[1-1/2(N_e)] + V_M
\end{gather*}
See ch7.r for the code to implement the recursive calculation for $V_{A(t)}$
\item The rate of mutation of lethal recessives is $2.6*10^{-6}$ per gene and generation and the frequency of such alleles in a population is $1.4*10^{-4}$. Are these estimates compatible with the complete recessivity of lethal alleles? If not, what would be the most likely value of the dominance coefficient h?\\
The maximum possible frequency of a deleterious allele at the deleterious mutation selection balance can be approximated by $\hat{q}^2 = u/s$. For recessive lethals, $s = 1$ and $u = \sqrt{q} = \sqrt{2.6*10^{-6}} = 0.00161$, so this estimate is not compatable with complete recessivity of lethal alleles because there is about an order of magnitude difference between expected and observed allele frequencies. To calculate the dominance coefficient, we use $h=u/q	= 0.00161/1.4*10^{-4} = 0.0186$
\item The following data correspond to fitness value of six individuals of a population of large size and the mean fitness of their progeny by self-fertilization. Is it possible to estimate the deleterious genomic mutation rate, U, the mean effect of the mutations $\bar{s}$ and its average dominance coefficient, $\bar{h}$? \\
Yes, if we assume that the population is at the deleterious mutation-selection equilibrium. See chp7.r for the code to answer this question.
\end{enumerate}

{\large \bf Self Assessment}
\begin{enumerate}
\item Multicellular eukaryotes generally have a higher rate of mutation per round of replication than prokaryotes or unicellular eukaryotes.\\
True, on page 156 there is a figure detailing the haploid mutation rate of E. coli and yeast as 0.0002 while for (multicellular) eukaryotes this value is 0.08.\\
\item The probability of fixation of a mutation with selective advantage $s = 0.01$ in a population of size $N = 10$ is approximately equal to that of a neutral mutation.\\
For a neutral mutation, the probability of fixation is $1/2N = 1/20 = 0.05$ With $s = 0.01$, and assuming $N_e = N = 10$, then the mutation is quasi-neutral since $-1 < |N_es = 10(0.01)=0.1<1$ with fixation probability $P_f \approx 1/2N + Ne_S/3N(1+h) = 1/20+10(0.01)/30(1+0.5) = 1/20+0.005 = 0.0505 \approx 0.05$, so this is true\\
\item The Bateman-Mukai method of estimation of the mutation rate and mean effect of mutations in mutation accumulation experiments will give overestimations of U and underestimates of the mean s if the effect if mutations is highly variable.\\
On page 155 it is stated that biases for these estimates can arise from the possibility of non-mutational changes in trait means occurring in experiments or produced by contaminations or adaptive changes (not necessarily due to highly variable mutations). Furthermore, in the preceding paragraph, it is stated that using a shape parameter of $\beta = 1$ we see overestimations of $\bar{s}$ and underestimations of U, so this is false.\\
\item Although an enzymatic mutation had additive gene action for enzyme activity, its effect on the flux of the metabolic pathway in which it participates may be recessive if the effect of the mutation is large.\\
True, a mutation with slight reduction in activity produces a heterozygote whose effect is intermediate between the homozygotes. Larger mutations such that the activity is 0 will cause the heterozygote to be more similar to the wild homozygote so the mutation now has recessive gene action because the other homozygote is being masked.\\
\item It has been estimated in e.coli and yeast that approximately 1 percent of mutations can be beneficial.\\
This is true and is stated on page 160. \\
\item The additive model implies independence of effects between loci for fitness.\\
False, the additive model implies a summation, but independence (multiplicative) is widely used. \\
\item For a purely neutral trait, the genetic variance of equilibrium increases indefinitely by mutation as the effect population size increases.\\
True, $V_A$ is proportional to $N_e$ and will increase (equation 7.7).\\
\item Welfare and medicine are expected to increase the mutation load in human populations.\\
Mutation load: decrease in mean fitness from the presence of segregating deleterious mutations. Load is not affected however, because it only depends on mutation rates, not the increase in equilibrium frequencies of deleterious alleles.\\
\item The absence of recombination is the reason why faster fixation of deleterious mutations occurs in species with asexual reproduction.\\
This is true and is known as Muller's ratchet.\\
\item Muller's ratchet can also occur in autogamous species.\\
False, because recombination in autogamy would prevent the accumulation of mutations in the same manner as for species which reproduce sexually.
\end{enumerate}


\medskip



\end{document}