\documentclass[12pt]{amsart}

\addtolength{\hoffset}{-2.25cm}
\addtolength{\textwidth}{4.5cm}
\addtolength{\voffset}{-2.5cm}
\addtolength{\textheight}{5cm}
\setlength{\parskip}{0pt}
\setlength{\parindent}{15pt}

\usepackage{amsthm}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage[colorlinks = true, linkcolor = black, citecolor = black, final]{hyperref}

\usepackage{graphicx}
\usepackage{multicol}
\usepackage{ marvosym }
\usepackage{wasysym}
\newcommand{\ds}{\displaystyle}


\pagestyle{myheadings}

\setlength{\parindent}{0in}

\pagestyle{empty}

\begin{document}

\thispagestyle{empty}

{\scshape VETM 690} \hfill {\scshape \Large  Ch. 5: Effective Population Size} \hfill {\scshape Feb 15 2021}
 
\medskip

\hrule

\bigskip

\bigskip

{\large \bf Methods of calculating $N_e$:}

\begin{itemize}
\item  Real populations may not meet the assumptions in the ideal population of the Wright and Fisher model
\item Effective population size $N_e$: the size that would have an ideal population that gives rise to drift or inbreeding observed in a real population. In other words: it translates the size of a real population into the size of an idealized population that shows the same effects (drift or inbreeding) as the population under study). 
\item One method to calculate $N_e$: find the largest non-unit eigenvalue of a transition matrix $P$ of a Markov chain. Calculate $N_e = 1/(2(1-\lambda))$
\item Can also use mutation effective size (defined from probability of identity in state of a gene under some mutation model with a defined rate)
\item Most common method: coalescence effective size. Coalescence time of two alleles is the number of generations that must be traced back in the genealogy to the ancestor from which they come. If we can estimate the mean time of coalescence T in a real population we can predict effective size $N_e = T/2$
\end{itemize}
{\large \bf Prediction of $N_e$ in populations with no selection:}
\begin{itemize}
\item The absence of self-fertilization means that the appearance of inbreeding is delayed by one generation, reducing its' rate of increase, where $N_e = N + 1/2$. This effect is negligible and will be ignored.
\item Difference number of males and females: $N_e = \frac{4N_mN_f}{N_m + N_f}$, where $N_m$ and $N_f$ are the number of individuals that can intervene in reproduction
\item The harmonic mean should be used whenever we average (effective) population sizes to predict their impact on dift or inbreeding
\end{itemize}

{\large \bf Variable population size across generations:}
\begin{itemize}
\item In a population where all other assumptions of the ideal population by Wright and Fisher are met, if the population has a variable size:  $1/N_e = 1/t\sum_{i=0}^{t-1}1/2N_i$ (the effective size is the harmonic mean of population sizes in each generation)  
\item Example (population bottleneck): if a population size goes from 2500, 2000, 50, 1500 to 3000 then effects generated by inbreeding and drift are high because they correspond to that of a population of $N = 228$. This is why genetic diversity in a population can be lower than what is expected at its' current size.
\item Variance and inbreeding effective sizes coincide but differ if population size changes
\item Drift of allele frequencies depends on the number of offspring. If genealogical or molecular marker data is available, variance effective size should be estimated from rate of increase in coancestry
\item Inbreeding in offspring depends on number of parents. If genealogical or marker data is available, inbreeding effective size should be estimated from rate of increase in inbreeding
\end{itemize}

{\large \bf Non-random contribution of parents to offspring:}
\begin{itemize}
\item The probability that two random gametes taken at random in a population come from the same individual: $(\sum k_i^2 - \sum k_i) / (2N(2N-1))$
\item Using variance of samping $k_i$ gametes, $N_e$ can be calculated as $4N / (2+S_K^2)$
\item Under the ideal population, the distribution of the number of gametes contributed by the parents is Poisson
\item These can be extended to where number of males and females differ  
\end{itemize}

{\large \bf Partial mating among relatives:}
\begin{itemize}
\item Hardy-Weinburg disequilibrium implies a decrease in the frequency of heterozygotes. In the previous calculation for $N_e$, this corresponds to the 2 term in the denominator (representing variation from Mendelian segregation), which gets reduced by a fraction of $\alpha$ from chapter 4 since heterozygotes contribute 2 alleles to offspring generating less variation in allele frequencies
\item The variance term (representing variation from different contributions from individuals) in the previous equation for $N_e$ is increased by a fraction of $alpha$, because homozygotes contribute a single allele to offspring generating more variation in allele frequencies 
\item Culminating in $N_e = 4N / (2(1-\alpha) + S_k^2(1+\alpha))$
\end{itemize}


{\large \bf Overlapping generations:}
\begin{itemize}
\item Generations can become imbricated creating a complex structure of ages with differential survivals and fecundities within an age class. 
\item With constant age structure, $N_e$ can be approximated as $4N_aI_g / (2 + S_K^2)$ where $N_a$ is the number of reproductive individuals that enter the population in each cohort and $I_g$ is the generational interval
\item Without a constant age structure, then if genealogical relationships are available the effective size can be calculated from rate of increase in inbreeding $N_e = 1/(2\bar{\Delta F})$
\end{itemize}

{\large \bf Prediction of $N_e$ in selected populations:}
\begin{itemize}
\item Genetic draft: frequency of a neutral allele carried by a successful breeder will increase in successive generations. These changes are more directional than those by drift. 
\item Selection reduces $N_e$, translating to increases in genetic drift and inbreeding
\item Linkage: linked loci decrease $N_e$
\item Background selection
\item Deleterious mutations: natural selection compensates for declines in fitness
\item Neutral gene: depends on frequency of recombination. As linkage between neutral and selective alleles increases, reduction of effective size is greater 
\end{itemize}

{\large \bf Prediction of $N_e$ in subdivided populations:}
\begin{itemize}
\item GO BACK AND REVIEW 
\end{itemize}

{\large \bf Applications of theory of $N_e$ to conservation:}
\begin{itemize}
\item $N_e$ provides a summary of the past history of a population in regards to inbreeding and drift
\item Mating procedures can be designed to eliminate the variance of parental contributions, leaving behind Mendelian segregation of heterozygotes as the only source of drift
\item Maker assisted selection can reduce variation from Mendelian segregation of heterozygotes too
\item Or, manipulate meiosis such that we use more than one gamete from a single meiosis

\end{itemize}

{\large \bf Problems}   
\begin{enumerate}
\item In problem 4.1 we considered a set of ideal populations of census size N = 20 individuals each. Suppose now that the populations are maintained with 4 males and 16 females each generation. What would be the expected values of the heterozygosity, the variance in the allele frequencies and the inbreeding at generations $t = 5, 20$ and $100$?
\begin{gather*}
N_e = \frac{4N_fN_m}{N_m + N_f} = \frac{4(4)16}{20} = 12.8\\
H_0 = 1 - \sum_{i=1}^np_i^2 = 1 - [(0.3)^2 + (0.7)^2] = 1 - 0.58 = 0.42 \\
F_t = 1 - (1 - \frac{1}{2N_e})^t \\
F_5 = 1 - (1 - \frac{1}{(2)12.8}) ^ 5 =  0.181\\
F_{20} = 1 - (1 - \frac{1}{(2)12.8}) ^ {20} = 0.549\\  
F_{100} = 1 - (1 - \frac{1}{(2)12.8}) ^ {100} = 0.981\\
\sigma _t^2 = p_0q_0F_t \\
\sigma_5^2 = 0.3 * 0.7 * 0.181  = 0.038\\
\sigma_{20}^2 = 0.3 * 0.7 * 0.549 = 0.115\\
\sigma_{100}^2 = 0.3 * 0.7 * 0.981 = 0.206\\
H_t = H_0(1 - \frac{1}{2N_e})^t \\
H_5 = 0.42[1 - (\frac{1}{(2)12.8})^5] = 0.344\\
H_{20} = 0.42[1 - (\frac{1}{(2)12.8})^{20}] = 0.189\\
H_{100} = 0.42[1 - (\frac{1}{(2)12.8})^{100}] = 0.008
\end{gather*}

\item In problem 4.2 we considered a plant species whose natural reproduction is a combination of autogamy and allogamy in equal proportions. From a large population of this species a line with census size $N = 20$ was founded, which was maintained for 10 generations in the greenhouse by allogamous reproduction. 
\begin{gather*}
1 - F_{IT} = (1-F_{IS})(1-F_{ST}) \\
S_K^2 \approx 2 + 2\beta = 2 + 2(0.5) = 3\\ 
N_e = \frac{4N}{2(1-\alpha) + S_K^2(1+\alpha)} = \frac{(4)20}{2(1-0.333) + 3(1+0.333)} = 15\\
F_{10} = 1 - (1-\frac{1}{2(15)})^10 = 0.288 \\
F_{IT} = 1 - (1-0.333)(1+0.288) = 0.525
\end{gather*}

\item In a population that is maintained with 16 males and 32 females in panmixia, what is the effective size corresponding to autosomal, X-linked and Y-linked loci?

\begin{gather*}
N_{autosomal} = \frac{4N_mN_f}{N_m + N_f} = \frac{4(16)32}{16 + 32} =  42.67\\
N_x = \frac{9N_mN_f}{2N_m + N_f} = \frac{9(16)32}{4(16) + 2(32)} = 36\\
N_y = N/2 = 8/2 = 4
\end{gather*}

\item Consider lines of size $N = 20$ of problem 5.2 which are reproduced with 50 percent autogamy, and calculate their effective size taking into account the effect of natural selection acting on deleterious mutations. Assume that the haploid genome rate of deleterious mutation is $U = 0.2$, that the effect of mutations is $s = 0.1$, constant for all of them, and that there is no linkage.\\
Variance of selective advantage: $C^2$. Cumulative term of selection: Q. Assume that selection does not reduce genetic variance so that $G = 1$. Lastly, correlation between selective advantages of parents (r) can be approximated by $\beta$ for selfing, which we know is 0.5.
\begin{gather*}
C^2 = Us = 0.2(0.1) = 0.02\\
Q = \frac{2}{2 - G(1+r)} = \frac{2}{2 - (1+0.5) = 4}\\
N_e = \frac{4N}{(2(1-\alpha) + (S_k^2 + 4Q^2C^2)(1+\alpha)} = \frac{4(20)}{(2(1-0.333) + (3 + 4(16)0.02)(1+0.333)} = 11.36 
\end{gather*}

\item Suppose a population is maintained with $N=16$ individuals that mate in pairs and the following numbers of descendants per pair are obtained: 1,3,4,0,,2,0,5, 1. (a) What would be the effect size of the population? (b) If the contribution of the pairs were equaized so that each couple contributed a male and female to the offspring, what would then be the effective size? What would it be if the contribution of four of the pairs were of two males and that of the other four pairs of two females?

\begin{gather*}
S_K^2 = Var(1,3,4,0,2,0,5,1) = 3.429\\
N_e = \frac{4N}{2 + S_K^2} = \frac{4(16)}{2 + 3.429} = 11.79\\
(b) S_K^2 = 0\\
N_e = \frac{63}{2} = 32
\end{gather*}

\end{enumerate}

{\large \bf Self Assessment}
\begin{enumerate}
\item The different types of effective size (variance, inbreeding, eigenvalue, coalescence, etc. usually coincide exactly or approximately in their asymptotic value.\\
True: the text states that except in very specific situations, the predictions of effective population size coincide, or only differ in second order terms.\\
\item To average different population sizes, the harmonic mean is usually used, because the effective size usually affects the denominator of the expressions in which it is found.\\
True, the text states that when we want to average (effective) population sizes for predicting their impact on drift or inbreeding, we use the harmonic mean, because small values have the most relevance and are in the denominator in these expressions. \\
\item In populations with a certain percentage of matings between relatives, the variance of the contributions of parents to progeny decreases with respect to that corresponding to a panmictic population.\\
False, Hardy-Weinberg disequilibrium implies a decrease of heterozygotes and increase of homozygoytes. The increase of homozygotes corresponds to an increase in the variation between parental contributions by a factor of $1 + \alpha$. This results in a greater variance in allele frequencies, because homozygotes only contribute one allele. \\
\item The effective size decreases as the generation interval increases in populations with overlapping generations.\\
False, effective size in overlapping generations is linearly proportional to generational interval $N_e \propto I_g$, so an increase in generational interval would increase $N_e$.\\
\item The magnitude of the genetic drift that affects the genes of the X chromosome in XX-XY species or the Z chromosome in ZZ-ZW species is 25 percent less than that of autosomal genes.\\
Assuming an equal number of males and females, then true. $N_e = 3N/4$. The expression of sex-linked genes is given greater weight in the heterogametic sex. This could be false? My answer deals with $N_e$, not drift.\\
\item The effective size referring to neutral genes is drastically reduced when there is linkage between these and other loci subjected to selection.\\
True, the cumulative effect of selection produced by mutations on neutral genes linked to genes that are selected can be approximated by $Q_c \approx \frac{1}{s+c}$. As linkage becomes greater, the percent recombination c increases which increases the cumulative effect.\\
\item The effective size of a subdivided population always increases with the differentiation in allele frequencies between sub-populations.\\
True, the effective size of a subdivided population $N_e = \frac{Nn}{1-F_{ST}}$. As differentiation increases (approaches 1), $N_e$ approaches infinity. Reasoning: there is little to no genetic drift because if sub-populations remain isolated, different allelic variants could become fixed in the different subpopulations without being lost, resulting in a high variance effective size. This scenario is only true if sub-populations contribute identically to the offspring in each generation. 
\item With equal contributions from parents to offspring, if mating between relatives is forced, the long-term effective size increases in comparison with the panmictic scenario.\\
True, the thinking is along the same lines as question 7. 
\item The demographic methods of estimating the effective size tend to produce underestimates, by not taking into account all possible sources of genetic drift in the population.\\
False, demographic methods have the advantage of incorporating sources like genetic drift and inbreeding. \\
\item The larger the effective population size, the larger is the expected linkage disequilibrium between two closely linked loci.\\
False, linkage and $N_e$ are inversely proportional as shown by: $N_e \approx N exp\frac{-U}{s + (L/2)}$, where L is the linkage in the chromosomal segment. So a larger $N_e$ must arise from lower $L$.
\end{enumerate}

\medskip



\end{document}